### ISSUE TITLE
copy image to local sometimes cannot work

### ISSUE THREAD

**Issue by qigtang** [typora/typora-issues#4525]
- [x] Searched existing issues to avoid creating duplicates.

- [x] Confirmed that it can be reproduced in built-in themes without customized css.

   If it only exists in 3rd party themes or css, you can still report it, but please attach the theme target or the css file. We may not "fix" it, if it is caused by 3rd party themes or css styles that we cannot support.

- [ ] Searched http://support.typora.io/

----

**Describe the bug**
In the preference setting, if I select "copy image to ./${filename}.assets", select online image and then copy most of time 
will work, but find some scenarios it cannot work. Please see reproduce step.

**To Reproduce**
1. Go to https://mp.weixin.qq.com/s/vDPYwzHRok6KFtYumMsa7g
2. copy the first image there, right mouse button, copy image, paste into one typora document, you will get

https://mmbiz.qpic.cn/mmbiz_png/xBVu2dcQn9xfI3GrojZdk5II3mQTnq87GHyQ7ejYkvZhSkibDEWVsGShMIUOKYiafcZsahWn2bM72FY3YebS6Gaw/640?wx_fmt=png&tp=webp&wxfrom=5&wx_lazy=1&wx_co=1

3. after that, "copying image" will be spinning for a while, but no image will be copied into assets folder.
4. 
**Expected behavior**
Should be able to copy it.

**Screenshots / Screencasts**
![image](https://user-images.githubusercontent.com/7813095/126369581-79bca0b8-8f64-4580-aa7f-617233e74f5e.png)

**Sample Markdown File**
not applicable.

**Desktop (please complete the following information):**
windows 10 64bit

**Typora Version**
e.g: 0.11.0

**Additional context**
Another suggestion regarding image naming. sorry I forgot the website that causing the trouble. If in a website, several images have the same basename, but different path, typora will copy it using the same basename, that means new copied image will overwrite old images, even if they are different images in original website. 
Suggestion: when copying images from website, maybe it is better to use sha1 hash of the contents of images as local file name, instead of original file name.

**Comment by abnerlee** [comment]
should be fixed in 0.11.2

**Comment by qigtang** [comment]
Great job! I tested with the case above, and confirm it is fixed.

Noticed one minor issue/difference:

![image](https://user-images.githubusercontent.com/7813095/126813788-c7c70fd0-db60-4efe-93be-a7a2595de3bf.png)

In preference, I already set "copy to local", but in typora, when I copy image, the new version will give me 3 options to choose.

![image](https://user-images.githubusercontent.com/7813095/126814064-d4019c3e-d03e-44de-bbaf-3092442a7a7c.png)

Which means either move mouse to pick one option, or  "down" key, press "enter". which will grab some attention from user, and have a minor impact on workflow. 

Suggestion: 
- If preference already set to copy to local, default behavior is copy to local.
- Or at least set the default selection of pull down manual to "copy to local”, then user only need to press enter to confirm, less attention grabbing UI design, it won't impact thought flow.
